libopenage/renderer/demo/demo_0.cpp

```diff
@@ -50,6 +51,10 @@ void renderer_demo_0(const util::Path &path) {
 		false,
 	};
 
+	if (!check_uniform_completeness({display_stuff})) {
```

Comment:
Instead of `!`, `&&`, and `||` we usually use `not`, `and`, and `or`, since they are more readable :)